app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -186,5 +147,31 @@ def experience_to_hash(experience)
         description: experience.details,
       }
     end
+
+    def qualifications
+      {
+        gcses: application_form.application_qualifications.gcses.map do |gcse| qualification_to_hash(gcse) end,
+        degrees: application_form.application_qualifications.degrees.map do |degree| qualification_to_hash(degree) end,
+        other_qualifications: application_form.application_qualifications.other.map do |other| qualification_to_hash(other) end,
+      }
+    end
+
+    def qualification_to_hash(qualification)
+      {
+        qualification_type: qualification.qualification_type,
+        subject: qualification.subject,
+        grade: "#{qualification.grade}#{' (Predicted)' if qualification.predicted_grade}",
+        award_year: qualification.award_year,
```

Comment:
the DB column is nullable but the JSON schema field is not (thought that wasn't part of this PR). Same applies to `qualification_type` and `subject`.

Reply:
good catch — one for the backlog?